### ISSUE TITLE
publishing a repository to GitHub shows as ongoing operation

### ISSUE THREAD

**Issue by ranjan-ranjan** [github/VisualStudio#589]
- GitHub Extension version: 2.0.15.1
- Visual Studio version: Microsoft Visual Studio Community 2015 Update 3

I have created a basic Console Application in C#. And added it to source control from the context menu from "Solution Explorer" Document window. So, now it is git repository. And then I publish it to GitHub. The files are visible in GitHub, but the result is this in Visual Studio:
![2](https://cloud.githubusercontent.com/assets/16834561/19259818/86d089fa-8fa2-11e6-8bd0-77cfb309f49a.png)

**Comment by grokys** [comment]
This indeed looks like a problem with the GitHub for Visual Studio extension. Could you provide the extension log file? You can find this at `%localappdata%\GitHubVisualStudio\extension.log`.

**Comment by ranjan-ranjan** [comment]
Here is the required log:


2016-10-10 19:54:30.6368|INFO|thread: 1|RepositoryHost|Log in from cache for login 'ranjan-ranjan' to host 'https://api.github.com/' SUCCEEDED
2016-10-10 20:12:00.1599|INFO|thread: 1|RepositoryHost|Logged off of host 'https://api.github.com/'
2016-10-10 20:12:00.1599|INFO|thread: 1|LoginCache|Erasing the git credential cache for host 'https://github.com/'
2016-10-10 20:12:12.2941|INFO|thread: 1|RepositoryHost|Log in from cache for login 'ranjan-ranjan' to host 'https://api.github.com/' SUCCEEDED
2016-10-10 20:12:12.2991|INFO|thread: 1|RepositoryHosts|Log in to GitHub.com host 'https://api.github.com/' with username 'ranjan-ranjan' SUCCEEDED
2016-10-11 10:58:10.8149|INFO|thread: 1|RepositoryHost|Log in from cache for login 'ranjan-ranjan' to host 'https://api.github.com/' SUCCEEDED
2016-10-11 10:58:10.8590|INFO|thread: 1|RepositoryHost|Log in from cache for login 'ranjan-ranjan' to host 'https://api.github.com/' SUCCEEDED

**Comment by grokys** [comment]
Hmm, well there's nothing in there. Does this happen every time you try to publish a repository, or was it just a one-off?

**Comment by shana** [comment]
The title you highlighted in the screenshot doesn't mean the repository is pushed, it just means it is a local git repository. Does your local git repository have a remote configured? You can check this at the git settings page.

**Comment by grokys** [comment]
@shana I assumed what he was highlighting was the progress bar at the top of the view, though I could be wrong - the bug report isn't clear.

@ranjan-ranjan could you describe what exactly you're highlighting in your screenshot?

**Comment by ranjan-ranjan** [comment]
@grokys https://github.com/github/VisualStudio/issues/589#issuecomment-252957052 - it happened only that time. I have tried to reproduce this issue again many times.

https://github.com/github/VisualStudio/issues/589#issuecomment-252965485 - yes the progress bar was still going on even when I could see the repository on GitHub.

@shana yes, the repository isn't published. it was a local solution. I have converted it to git repository using context menu from "Solution Explorer" Document window ("Add to source control" option).
